.travis.yml

```diff
@@ -18,4 +18,8 @@ script:
     --exclude ../kubespray/
     --exclude idr-kubernetes-apply.yml
     --exclude idr-reset-users-groups.yml
-  - cd ansible; molecule test
+  - cd ansible; molecule test -s $SCENARIO
+
+env:
+  - SCENARIO=publicidr
```

Comment:
It's been a while we did not come to a naming discussion. I am still very unconvinced by the usage of `publicidr`, minimally because e.g. `idr-analysis` is similarly public. If we are considering any of these are components of the overarching IDR deployment, an alternate proposal for the types would be something of  `ingress/datasets/analysis` as a way to represent the aim of each infrastructure.

Reply:
I thought this might come up 😂. I'm just using the names from https://github.com/IDR/deployment/blob/IDR-0.6.0/ansible/openstack-create-infrastructure.yml

Reply:
:+1: for changing. We have enough external confusion with "IDR" anyway.

Reply:
Indeed 😄 . And at the time the split was introduced, we didn't manage to get to a better naming consensus but knew we might eventually come back to it - see https://github.com/IDR/deployment/pull/95#pullrequestreview-106083611. Happy to consider other alternatives obviously.

Reply:
How about get this PR in, then whoever's brave enough to make a proposal can open a renaming PR?

Reply:
I am up for have this integrated together with #155 so that the new addition is tested by `molecule`. Regarding the new variable name

https://github.com/IDR/deployment/blob/94876d81a84b1ed48515ff5e163fbb0cde5762bb/ansible/openstack-create-infrastructure.yml#L4-L5

Is there an easy Jinja/Ansible way to check 2 variables (i.e. maintain backwards compatibility)? Or is the only way to have some kind of hard break?

Reply:
We could, but since we're fully in control of this repository I think adding logic to handle alternative variable names makes maintenance more complex for no real benefit.